test/file.ts

```diff
@@ -4118,7 +4118,7 @@ describe('File', () => {
           await file.save(DATA, options);
           throw Error('unreachable');
         } catch (e) {
-          assert.strictEqual(e.message, 'first error');
+          assert.strictEqual(e.message, 'unreachable');
```

Comment:
We shouldn't test this like this. If we're checking that it retries, we shouldn't throw the "unreachable" error (because it will be reachable) and instead assert the retryCount